### DISCUSSION TITLE
install zigbee2mqtt on True Nass

### DISCUSSION THREAD

**Discussion by tartuf9** [Koenkk/zigbee2mqtt#9957] [General]
hello everyone, I am a beginner and I want to install zigbee2mqtt on my TrueNas server. its been over 1 week since i'm looking and i couldn't find anything if there was someone who could help me. thank you

**Comment by lux73** [comment]
https://www.truenas.com/community/search/2211421/?q=zigbee2mqtt&o=relevance

did you search the TrueNAS Forum? There are some Threads relating to z2m

**Comment by tartuf9** [comment]
hello and thanks for your help, i looked on the truenas site and i cant find any real help, i will ask questions to the person posting the comments. Otherwise I will have to find another solution. thank you for the moment and if you have an idea I remain at your disposal.
